pyspark/bigdl/keras/backend.py

```diff
@@ -20,18 +20,21 @@
 from bigdl.keras.optimization import *
 from bigdl.util.common import get_spark_context
 from bigdl.util.common import to_sample_rdd
+from bigdl.util.common import redire_spark_logs, show_bigdl_info_logs
 
 
-class KerasModelWrapper():
+class KerasModelWrapper:
 
     def __init__(self, kmodel):
+        redire_spark_logs()
+        show_bigdl_info_logs()
         self.bmodel = DefinitionLoader.from_kmodel(kmodel)
         WeightLoader.load_weights_from_kmodel(self.bmodel, kmodel)  # share the same weight.
         self.criterion = OptimConverter.to_bigdl_criterion(kmodel.loss)
         self.optim_method = OptimConverter.to_bigdl_optim_method(kmodel.optimizer)
         self.metrics = OptimConverter.to_bigdl_metrics(kmodel.metrics) if kmodel.metrics else None
 
-    def evaluate(self, x, y, batch_size=32, sample_weight=None, is_distributed=False):
+    def evaluate(self, x, y, batch_size=32, sample_weight=None, is_distributed=True):
```

Comment:
any reason for  this?

Reply:
I think default using distributed fashion is better?
1. For BigDL, our feature towards Keras is that we can do training in a distributed way.
2. Seems that distributed is better supported. For example, evaluation/validation doesn't support local mode. If user use the default false, he/she will get an error.

Reply:
Ideally, we should be able to run keras code without changes. so let's turn the default value to be false to meet that requirement.  We can fix the not supported error in a separated PR.

Reply:
OK. I will fix it.